src/components/Amalgamation/ResultingBusinessName.vue

```diff
@@ -36,7 +36,7 @@
               :businessId="getBusinessId"
               :companyName="companyName"
               :correctionNameChoices="correctionNameChoices"
-              :entityType="null"
+              :entityType="getEntityType"
```

Comment:
Need to pass this down to shared CorrectName component so it can pre-format the numbered name if needed (eg, "B.C. COMMUNITY CONTRIBUTION COMPANY LTD.").